tests/fixtures/saas/alchemer_fixtures.py

```diff
@@ -0,0 +1,96 @@
+import random
+import string
+import time
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+import requests
+
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("alchemer")
+
+
+@pytest.fixture(scope="session")
+def alchemer_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain": pydash.get(saas_config, "alchemer.domain") or secrets["domain"],
+        "api_token": pydash.get(saas_config, "alchemer.api_token")
+        or secrets["api_token"],
+        "api_token_secret": pydash.get(saas_config, "alchemer.api_token_secret")
+        or secrets["api_token_secret"],
```

Comment:
Make sure to update this once you change it to `api_key` and `api_key_secret` in the connector params